frontend/src/scenes/surveys/surveyViewViz.tsx

```diff
@@ -133,33 +134,48 @@ function StackedBar({ segments }: { segments: StackedBarSegment[] }): JSX.Elemen
 }
 
 export function UsersCount({ surveyUserStats }: { surveyUserStats: SurveyUserStats }): JSX.Element {
-    const { seen, dismissed, sent } = surveyUserStats
-    const total = seen + dismissed + sent
+    const { featureFlags } = useValues(featureFlagLogic)
+    const { seen, dismissed, completed, partial } = surveyUserStats
+    const total = seen + dismissed + completed + partial
+
+    const isPartialResponsesEnabled = featureFlags[FEATURE_FLAGS.SURVEYS_PARTIAL_RESPONSES]
```

Comment:
this flag isnt created yet, so do not merge this before creating the flag (team 2 initially?)